packages/core/src/messages/responses/cache-set-if-absent-or-equal.ts

```diff
@@ -9,15 +9,15 @@ interface IResponse {
 /**
  * Indicates the new value was set because the key did not exist or the existing item was equal to the supplied `equal` value.
  */
-export class Stored extends BaseResponseSuccess implements IResponse {
+export class Stored extends ResponseBase implements IResponse {
```

Comment:
just wondering what was the logic here, I could go both ways on this, cause techincally `Stored` and `NotStored` are successful api calls

Reply:
The Hit classes also extend ResponseBase instead of BaseResponseSuccess, so I thought only the classes named Success should extend that one. 
Should they all go the other way instead and mostly extend BaseResponseSuccess?

Reply:
I don't think it entirely matters honestly I think this `SuccessBase` class is mostly used for us to have `DRY` integration tests. I think that makes sense to me that if `Hit` doesnt extend `Success`, then `Stored` should also not extend `Success`